dev-guide/cluster-version-operator/dev/clusteroperator.md

```diff
@@ -35,9 +35,9 @@ There are 2 important things that need to be set in the ClusterOperator Custom R
 - `.metadata.name`: name for finding the live instance
 - `.status.versions[name=operator].version`: this is the version that the operator is expected to report. ClusterVersionOperator only respects the `.status.conditions` from instances that report their version.
 
-Additionally you might choose to include some fundamental relatedObjects.
+Additionally, you should include some fundamental [related objects](#related-objects) of your operator. 
```

Comment:
Is it worth mentioning that we have [payload informing jobs](https://github.com/openshift/origin/blob/master/test/extended/operators/clusteroperators.go#L59-L83) that ensure your component populates related objects for your cluster operator?

Reply:
Done!